### PR TITLE
Feature/add entries in ary query

### PR DESCRIPTION
**Pull request by Rup-Narayan-Rajbanshi**
## Changes
- Add lead entry information in assessment registry query
-  Modified lead query and filter to accept new assessment registry.
- Fix issue for not accepting empty list in multiple select imput.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

**Review by thenav56 — APPROVED** [review]

**Review by thenav56 — DISMISSED** [review]

File: apps/assessment_registry/migrations/0001_initial.py
Comment (thenav56) [inline apps/assessment_registry/migrations/0001_initial.py:1]: @Rup-Narayan-Rajbanshi 
We can also merge/reset migrations before merging the project branch to develop as well.

File: apps/assessment_registry/models.py
Comment (thenav56) [inline apps/assessment_registry/models.py]: Maybe we should use default=list instead.
Basically to avoid storing both null and [] as an empty value in the database.

File: apps/lead/tests/test_schemas.py
Comment (thenav56) [inline apps/lead/tests/test_schemas.py:275]: Can't we fix this instead?

File: apps/project/schema.py
Comment (thenav56) [inline apps/project/schema.py:427]: Not required anymore? Or we have moved this?

File: apps/static/image/graphQL-logo.svg
Comment (thenav56) [inline apps/static/image/graphQL-logo.svg:1]: Let's rebase the project branch with develop first. There are changes from develop.

**Review by thenav56 — CHANGES_REQUESTED** [review]

File: apps/static/image/graphQL-logo.svg
Comment (Rup-Narayan-Rajbanshi) [reply apps/static/image/graphQL-logo.svg:1]: > Let's rebase the project branch with develop first. There are changes from develop.

@thenav56 
I rebased project/assessment-registry with current develop and tried to push, but I am getting the following error. May be we dont have push access to the protected branch.
image

File: apps/lead/tests/test_schemas.py
Comment (Rup-Narayan-Rajbanshi) [reply apps/lead/tests/test_schemas.py:275]: @thenav56 , I have created Test Factories required for assessment registry in next PR https://github.com/the-deep/server/pull/1324. So this case will be handled in that PR.

File: apps/project/schema.py
Comment (Rup-Narayan-Rajbanshi) [reply apps/project/schema.py:427]: @thenav56 , this is removed in the next pr https://github.com/the-deep/server/pull/1324

**Review by thenav56 — APPROVED** [review]

**Pull request merged**
